ndcube/mixins/plotting.py

```diff
@@ -348,10 +349,10 @@ def _animate_cube_1D(self, plot_axis_index=-1, axes_coordinates=None,
         # Get real world axis values along axis to be plotted and enter into axes_ranges kwarg.
         if axes_coordinates[plot_axis_index] is None:
             xname = self.world_axis_physical_types[plot_axis_index]
-            xdata = self.axis_world_coords(plot_axis_index)
+            xdata = self.axis_world_coords(plot_axis_index, edges=True)
         elif isinstance(axes_coordinates[plot_axis_index], str):
             xname = axes_coordinates[plot_axis_index]
-            xdata = self.extra_coords[xname]["value"]
+            xdata = _get_extra_coord_edges(self.extra_coords[xname]["value"])
```

Comment:
I am wondering why the tests are failing for only `_animate_cube_1D` inspite of the fact that the new SunPy `plotting` now requires `pixel_edges` instead of `pixel_values`?

Reply:
Now, I know. 
All these days I was testing my code with `SunPy0.9.8` which was showing all these failing tests only for `1D` plots.
Now with `SunPy1.0` around, more tests for **>1D** are failing.

Reply:
I have corrected tests and edited that plotting takes place for `pixel_edges` instead of `pixel_values`. This covers most of the >1D cases.